tool/tctl/common/auth_command.go

```diff
@@ -757,6 +757,12 @@ client_encryption_options:
    keyfile: /path/to/{{.output}}.key
    truststore:  /path/to/{{.output}}.cas
    require_client_auth: True
+
+To enable x509 certificate authentication on you Scylla server (available from
+Scylla version 5.4.0), add the following to your scylla.yaml configuration file:
+
+authenticator: com.scylladb.auth.CertificateAuthenticator
+auth_certificate_role_query: CN=([^,\s]+)
```

Comment:
I see the official doc uses `auth_certificate_role_queries`, do both works?

Reply:
Also wonder what the `,` does in the regex, if `--db-user` is `abc,def`, only `abc` will be mapped?

Reply:
I can't find this option in scylla.yaml - should it have been `auth_certificate_role_queries`? Based off of this page: https://opensource.docs.scylladb.com/stable/operating-scylla/security/certificate-authentication.html#

Reply:
Yes, the option is `auth_certificate_role_queries`, I'll update it.

Reply:
> Also wonder what the , does in the regex, if --db-user is abc,def, only abc will be mapped?

From my tests: If `--db-user=alice,bob`, Scylla returns an error complaining that the role could not be found (given that the space is not present). With a space included (for example, `--db-user=alice\ bob`), only the first role/user is assumed.

Reply:
Ya. I would prefer something like `CN=(.+)` to make sure Scylla just takes everything in the CN. Are commas/spaces valid characters in a role name in Scylla?

Reply:
`[^,\s]+` should not match comma nor space, so it's interesting that it works with a space but not a comma